sale_tier_validation/views/sale_order_view.xml

```diff
@@ -2,6 +2,26 @@
 <!-- Copyright 2019 Open Source Integrators
      License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
 <odoo>
+<record id="view_order_form" model="ir.ui.view">
+        <field name="name">sale.order.form</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//button[@name='action_confirm'][1]" position="attributes">
+                <attribute
+                    name="attrs"
+                >{'invisible':['|', ('validation_status', 'in', ('pending', 'rejected')),('state', '!=', 'sent')]}</attribute>
```

Comment:
Any other module changing attrs will invalidate this. At least this should be said in the ROADMAP.